spec/factories/submission.rb

```diff
@@ -26,8 +26,8 @@
 
     factory :submission_with_validated_entries do
       after(:create) do |submission, _evaluator|
-        create_list(:invoice_entry, 2, :valid, submission: submission)
-        create_list(:order_entry, 1, :valid, submission: submission)
+        create_list(:invoice_entry, 2, :valid, submission: submission, total_value: 10.00)
```

Comment:
this makes me think that the factories for `SubmissionEntry` should perhaps be updated to set the total_value on validated entries. perhaps the legal framework traits should do so also to make the generated instance consistent with what we should see in reality.

Reply:
Yes, I'm afraid I avoided the second yak.